src/run-tests.js

```diff
@@ -8,6 +8,7 @@ import "./translator"
 import "./args"
 import "./text"
 import "./generator"
+import "./main"
```

Comment:
Since `main` imports everything else that's used in prod, I wonder if these other imports (lines 6–10) can be removed now.